### PR TITLE
flask mandril, email confirm

### PR DESCRIPTION
**Pull request by mjhea0**

### CODE REVIEW

**Comment by mjhea0** [comment]
WIP - do not merge into develop since this changes the user signup functionality

File: src/web/web.py [hunk 1]
```diff
@@ -28,10 +28,15 @@
 from flask import Flask
 from flask import render_template
 
+# Flask extensions
+from flaskext.mail import Mail
```
Comment (madflojo) [inline src/web/web.py]: web_1           | -------------------------------------------------
web_1           | Traceback (most recent call last):
web_1           |   File "/code/web.py", line 32, in <module>
web_1           |     from flaskext.mail import Mail
web_1           | ImportError: No module named flaskext.mail

Fixed it in my local instance.

**Comment by madflojo** [comment]
Hey, 

So when testing I got an error around the view "auth.confirm" which I found referenced here. 

https://github.com/mjhea0/cloudroutes-service/blob/develop/src/web/templates/user/confirm.html

Looks like auth.confirm isn't defined.

File: src/web/web.py (hunk 1 above)
Comment (mjhea0) [inline src/web/web.py]: Thanks!

**Comment by madflojo** [comment]
Closing this as these changes will require a hefty update before merged.

**Pull request closed without merging**